core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -17,6 +17,48 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
 
   }
 
+  "Snowflake Alter commands" should {
+
+    "transpile Single ADD Columns" in {
+      "ALTER TABLE t1 ADD COLUMN c1 INTEGER;" transpilesTo (
```

Comment:
```suggestion
    "ALTER TABLE t1 ADD COLUMN c1 INTEGER" in {
      "ALTER TABLE t1 ADD COLUMN c1 INTEGER" transpilesTo (
```
use the same name for in block as the source query. we'll add more syntactic sugar later

Reply:
Done